style.css

```diff
@@ -525,6 +634,114 @@ body {
     grid-area: 2/3/ span 1/ span 1;
   }
 
+  .pop {
+    position: fixed;
+    height: 100vh;
+    top: 0;
+    display: flex;
+    justify-content: center;
+    background-color: gray;
+  }
+
+  .popFlex {
+    background-color: #fff;
+    width: 75%;
+    height: 75vh;
```

Comment:
- [ ] Your popup container still doesn't adjust to fit its content as suggested by the previous reviewer. The previous reviewer gave you a suggestion that can guide you which actually helps but then you will have to remove the fixed height of 75vh so that the container can fit its content.

![image](https://github.com/Dachrono/Andy_Portfolio/assets/63473584/c2703756-6d47-4dd3-b814-6d1f36ee539a)

Reply:
Solved